region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -847,6 +847,24 @@
         "Freezing the lower one may be made easier or damageless by scrolling the camera when the Beetom is in the correct position and freezing it with a hero shot from the right."
       ]
     },
+    {
+      "link": [3, 3],
+      "name": "G-Mode Setup - Get Hit By Zebbo",
+      "requires": [
+        "Morph",
+        {"or": [
+          "canTrickyJump",
+          "Ice"
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithGModeSetup": {}
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}],
+      "flashSuitChecked": true,
+      "note": "Carefully lure the Beetom below the platform, then lure a Zebbo to the left door. Be careful not to let the Zebbo go off camera, or it will despawn.",
```

Comment:
I thought this meant bring the beetom with you to the door.  But it says to trap it under a floating platform.